### DISCUSSION TITLE
Make input delay configurable

### DISCUSSION THREAD

**Discussion by matt-bader** [orchidjs/tom-select#699] [General]
When typing in a TomSelect, there is a constant delay (around 250ms) between the keypress and the time that it is processed. If within this time another keypress occurs, the first one is not processed. Is it possible to shorten this delay or to disable this behavior entirely? 

In a LAN environment, I'd be happy to have a fast response while accepting the resulting increase in server load.

**Comment by pereorga** [comment]
See https://github.com/orchidjs/tom-select/issues/734
